SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -119,17 +120,34 @@ SCDocHTMLRenderer {
 			baseDir = baseDir ++ "/..";
 		};
 
+		thisIsTheMainHelpFile = (doc.title=="Help") and:
+			{((thisProcess.platform.name===\windows) and: (folder=="Help")) or: {folder==""}};
+
+		stream
+		<< "<!doctype html>"
+		<< "<html lang='en'>"
+		<< "<head><title>";
+
+		if(thisIsTheMainHelpFile) {
```

Comment:
i'll save that for another PR if you don't mind. i'm just trying to fix `<title>` here :)